deploy/olm-catalog/cluster-network-addons/0.0.1/cluster-network-addons-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kubevirt-cluster-network-addons-operator.v0.0.1
```

Comment:
Why do we use this versioning? why woudn't we stick with versioning to the cluster-network-addons-operator? Other users (HCO) of this, should use manifests released for a specific version anyway. Not sure how to version master.

Reply:
Does the cluster-network-addons-operator have a version (in source) that I could use instead?

In the short term this value isn't incredibly important (what number is there), but as you start releasing new versions  of the CSV (and make use of OLMs concept of channels) this will quickly become important.

Reply:
> Does the cluster-network-addons-operator have a version (in source) that I could use instead?

I had looked through trying to find one but could not.